utils/configuration_test.go

```diff
@@ -234,28 +244,27 @@ func TestParseStorageWithEnvironmentVariables(t *testing.T) {
 // If TLS is required and the parameters are missing, an error is returned
 func TestParseTLSNoTLSWhenRequired(t *testing.T) {
 	invalids := []string{
-		`{"server": {"tls_cert_file": "path/to/cert"}}`,
-		`{"server": {"tls_key_file": "path/to/key"}}`,
+		fmt.Sprintf(`{"server": {"tls_cert_file": "%s"}}`, Cert),
+		fmt.Sprintf(`{"server": {"tls_key_file": "%s"}}`, Key),
 	}
 	for _, configJSON := range invalids {
 		_, err := ParseServerTLS(configure(configJSON), true)
 		assert.Error(t, err)
-		assert.Contains(t, err.Error(),
-			"both the TLS certificate and key are mandatory")
+		assert.Contains(t, err.Error(), "no such file or directory")
 	}
 }
 
 // If TLS is not and the cert/key are partially provided, an error is returned
```

Comment:
nit: while you're in this file, can we update this comment to start with `// If TLS is not required`?